src/Main.java

```diff
@@ -60,23 +60,23 @@ public static void main(String args[]) throws ParseException {
         manager.Manager(hotel);
 
```

Comment:
Try separating code blocks in the main method by writing comments in between blocks of code doing one thing or testing a functionality.

It's too hard to read this way.